### PR TITLE
Make click catchers invisible

### PR DESCRIPTION
**Pull request by Kegdo**
## About The Pull Request

after testing, I figured out that CC actually HAVE a (shitty) use, so to NOT make me experience PTSD I just turn them invisible to human eye
literally one line change btw
## Why It's Good For The Game
honestly we could just remove CC one day
## Changelog
:cl:
code: Click catchers are invisible now. One is right behind your back.
/:cl:

### CODE REVIEW

**Comment by Kegdo** [comment]
WRONG FILE

**Comment by Kegdo** [comment]
AGAIN

File: code/_onclick/click.dm [hunk 1]
```diff
@@ -380,6 +380,10 @@ GLOBAL_LIST_INIT(click_catchers, create_click_catcher())
 /obj/screen/click_catcher/Destroy()
 	return QDEL_HINT_LETMELIVE
 
+/obj/screen/click_catcher/New()
```
Comment (TheShown911) [inline code/_onclick/click.dm:383]: I am pretty sure you need to keep arguments when making a subtype of a proc.

Comment (Kegdo) [reply code/_onclick/click.dm:383]: there is ..()?

Comment (SirRichardFrancis) [reply code/_onclick/click.dm:383]: > there is ..()?

/obj/screen/click_catcher/New(_name = "", mob/living/_parentmob, _icon, _icon_state)

File: code/_onclick/click.dm [hunk 2]
```diff
@@ -380,6 +380,8 @@ GLOBAL_LIST_INIT(click_catchers, create_click_catcher())
 /obj/screen/click_catcher/Destroy()
 	return QDEL_HINT_LETMELIVE
 
+/obj/screen/click_catcher/New(_name = "", mob/living/_parentmob, _icon, _icon_state)
+
```
Comment (SirRichardFrancis) [inline code/_onclick/click.dm:384]:
```suggestion
..()

```

**Review by SirRichardFrancis — CHANGES_REQUESTED** [review]

File: code/_onclick/click.dm (hunk 2 above)
Comment (Kegdo) [reply code/_onclick/click.dm:384]: why the fuck is it like that

File: code/_onclick/click.dm [hunk 3]
```diff
@@ -380,6 +380,9 @@ GLOBAL_LIST_INIT(click_catchers, create_click_catcher())
 /obj/screen/click_catcher/Destroy()
 	return QDEL_HINT_LETMELIVE
 
+/obj/screen/click_catcher/New(_name = "", mob/living/_parentmob, _icon, _icon_state)
+..()
```
Comment (SirRichardFrancis) [inline code/_onclick/click.dm:384]: ))))
```suggestion
    ..()
```

Comment (Kegdo) [reply code/_onclick/click.dm:384]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

**Comment by hyperioo** [comment]
What does this PR do exactly? Hide click catcher from the list when you right click a tile?

**Comment by Kegdo** [comment]
ye

**Review by hyperioo — APPROVED** [review]
Seems alright.

**Comment by drexample** [comment]
Where's my thank you? :^)

**Pull request merged**